### ISSUE TITLE
VIRUS!!!

### ISSUE THREAD

**Issue by NuxLO49** [Saenka/LOIC#1]
![Screenshot (6122)](https://github.com/Saenka/LOIC/assets/144965990/56ba7215-87ce-4866-acd9-e1624d4b6adf)

**Comment by Saenka** [comment]
false positive, the detection show hacktool , (loic is ddos/hack tool)

**Comment by SEAZ1** [comment]
Bullshit, its a trojan

**Comment by SEAZ1** [comment]
little fucker

**Comment by SEAZ1** [comment]
https://app.any.run/tasks/f6891351-b407-439e-8834-e56f065bebd3/

**Comment by SEAZ1** [comment]
shut the fuck up. You destroyed my Testing VM.

**Comment by Saenka** [comment]
i dont know why you aggressive to me? the any.run sandbox shows no weird connection or trojan and only LOIC.exe is run so its not binded or anything

**Comment by SEAZ1** [comment]
The task involved the execution of two processes: LOIC.exe and wmpnscfg.exe. LOIC.exe was located in a temporary folder and had a parent process with the PID 2412. wmpnscfg.exe was located in the Windows Media Player folder and had a parent process with the PID 1388. The task also included modifications to various files and registry keys.

The most interesting event from this task is the modification of the registry key "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Internet Settings\ZoneMap". The value of the "ProxyBypass" name was changed to "1". This indicates that the task may have attempted to bypass proxy settings, potentially for malicious purposes.

Another interesting event is the modification of the registry key "HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Internet Settings\Connections". The value of the "SavedLegacySettings" name was changed to a long hexadecimal string. This could indicate an attempt to modify or manipulate internet settings.

In conclusion, this task involved the execution of two processes and modifications to files and registry keys. The most interesting points for a malware analyst are the modification of the "ProxyBypass" value in the ZoneMap registry key and the modification of the "SavedLegacySettings" value in the Connections registry key. These actions suggest potential attempts to bypass proxy settings and manipulate internet settings, which could be indicative of malicious activity.
